tests/test_helpers.py

```diff
@@ -288,75 +275,28 @@ def test_template_escaping(self, app, req_ctx):
         assert rv == '<a ng-data=\'{"x": ["foo", "bar", "baz\\u0027"]}\'></a>'
 
     def test_json_customization(self, app, client):
-        class X(object):
```

Comment:
Removing this code changes these tests, they're specifically checking if `flask.json.loads` picks up the request context and blueprint overrides. Or am I missing something?

Some of these other tests can probably be removed though, as they should be tested in Werkzeug.